package.json

```diff
@@ -1,16 +1,21 @@
 {
   "name": "@yext/chat-ui-react",
-  "version": "0.8.2",
+  "version": "0.8.3",
   "description": "A library of React Components for powering Yext Chat integrations.",
   "author": "clippy@yext.com",
   "main": "./lib/commonjs/src/index.js",
   "module": "./lib/esm/src/index.mjs",
-  "types": "./lib/esm/src/index.d.ts",
   "sideEffects": false,
   "exports": {
     ".": {
-      "import": "./lib/esm/src/index.mjs",
-      "require": "./lib/commonjs/src/index.js"
+      "import": {
+        "default": "./lib/esm/src/index.mjs",
```

Comment:
nit: `default` key should be last in the object -- I believe order technically matter here: https://publint.dev/@yext/chat-ui-react@0.8.3-beta.1

Reply:
wtf :o today I learned the json order matters in package.json